src/PaddedViews.jl

```diff
@@ -309,4 +309,14 @@ function Base.showarg(io::IO, A::PaddedView, toplevel)
     toplevel && print(io, " with eltype ", eltype(A))
 end
 
+# The fallbacks work, but this is more efficient
+# TODO use use short-circuit evaluation,
+Base.any(f::Function, A::PaddedView) = f(A.fillvalue) | any(f, parent(A))
+Base.all(f::Function, A::PaddedView) = f(A.fillvalue) & all(f, parent(A))
```

Comment:
```julia
julia> a = fill(true, 2, 2)
2×2 Array{Bool,2}:
 1  1
 1  1

julia> A = PaddedView(false, a, axes(a))
2×2 PaddedView(false, ::Array{Bool,2}, (Base.OneTo(2), Base.OneTo(2))) with eltype Bool:
 1  1
 1  1

julia> all(identity, A)
false

julia> all(identity, collect(A))
true
```

Reply:
@goretkin, I pointed this problem in https://github.com/JuliaLang/julia/issues/35563#issuecomment-619682768.

Reply:
Similarly, there is a problem with cropping.
I have a sense of foreboding about the combination of those and `dims`.:fearful:

Reply:
@kimikage Yeah, I am not sure it is worthwhile to implement anything clever for `dims`. The required reasoning would be substantially more complex, and I can imagine it being slower for many cases where there isn't that much padding.

Not sure what problem with cropping you mean.

Reply:
```julia
julia> a = Bool[0 0; 1 1]
2×2 Array{Bool,2}:
 0  0
 1  1

julia> A = PaddedView(false, a, (1,3))
1×3 PaddedView(false, ::Array{Bool,2}, (Base.OneTo(1), Base.OneTo(3))) with eltype Bool:
 0  0  0
```

Reply:
Thanks for the example! Yeah, that would also need to be handled for sure.